background.js

```diff
@@ -88,24 +88,24 @@ chrome.runtime.onMessage.addListener(function (command) {
       update_acc = "u/" + account_num;
     }
 
-    // Google calendar
+    // Google Calendar
     const calendar_regex = "https:\/\/calendar.google.com\/calendar\/u\/[0-9].*";
     console.log(current_url, calendar_regex);
     if (current_url.match(calendar_regex)) {
       update_url_regex = RegExp("u\/[0-9]");
       update_acc = "u/" + account_num;
     }
 
-    // Google shopping
-    const shopping_regex = "https:\/\/www.google.co.in\/.*\?authuser=[0-9].*";
+    // Google
+    const shopping_regex = "https:\/\/www.google.com\/.*\?authuser=[0-9].*";
     console.log(current_url, shopping_regex);
     if (current_url.match(shopping_regex)) {
       update_url_regex = RegExp("authuser=[0-9]");
       update_acc = "authuser=" + account_num;
     }
 
-    // Google translate
-    const translate_regex = "https:\/\/translate.google.co.in\/.*\?authuser=[0-9].*";
+    // Google Translate
+    const translate_regex = "https:\/\/translate.google.com\/.*\?authuser=[0-9].*";
```

Comment:
Can you please add compatibility of regions?
You can search for regex which accepts either .com
or other regional domain endings

Should most likely be a well known regex

Reply:
```js
	const google_regex_type1 = ".*.google.*\/u\/[0-9].*"
    if (current_url.match(google_regex_type1)) {
      update_url_regex = RegExp("u\/[0-9]");
      update_acc = "u/" + account_num;
    }

    const google_regex_type2 = ".*.google.*\?authuser=[0-9].*"
    if (current_url.match(google_regex_type2)) {
      update_url_regex = RegExp("authuser=[0-9]");
      update_acc = "authuser=" + account_num;
    }
```

How about simplifying all codes to these two patterns? @anshulahuja98

Reply:
I think that makes sense
Please go ahead and make the changes for this.